lib/pbench/agent/base.py

```diff
@@ -104,3 +102,15 @@ def get_path(self, path):
             return pathlib.Path(path)
         else:
             return path
+
+    def verify_tool_group(self, group):
+        """Ensure we have a tools group directory to work with"""
+        self.tool_group_dir = self.pbench_run / f"tools-v1-{group}"
+        if not self.tool_group_dir.exists():
+            click.secho(
+                f'\t{self.name}: invalid option ("{group}"), directory not found: {self.tool_group_dir}'
```

Comment:
Can we modify this message to read, "... invalid --group option ..."?

Reply:
fixed